<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditProfileDialog</class>
 <widget class="QWidget" name="EditProfileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>1051</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Profile</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="DialogItemsGroupBox">
     <layout class="QVBoxLayout" name="DialogItemsGroupBox_3">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="m_StatusLabel">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_DoneButton">
        <property name="text">
         <string>Done</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="m_GreetingLabel">
        <property name="text">
         <string>Greeting:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="greetingEdit">
        <property name="text">
         <string>Hi There!</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="aboutMeLabel">
        <property name="text">
         <string>About Me:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="aboutMeEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="plainText">
         <string>I'm Wonderfull!</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="pictureOfMeLabel">
        <property name="text">
         <string>My Picture</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="browsePictureButton">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="takeSnapshotButton">
          <property name="text">
           <string>Take Snapshot</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="pictureOfMeFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="favoriteWebsitesLabel">
        <property name="text">
         <string>My Favorite Websites</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="favoriteWebsite1Edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="favoriteWebsite2Edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="favoriteWebsite3Edit">
        <property name="text">
         <string>http://www.gotvp2p.com</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../gotvcommon.qrc"/>
 </resources>
 <connections/>
</ui>
